3.1_task.py

```diff
@@ -0,0 +1,5 @@
+my_elements = [1, 2, 3, 4, 5, 6, 7, 8]
+for index in range(0, len(my_elements), 2):
+    print([index, my_elements[index]])
+for index in range(1, len(my_elements), 2):
+    print([index, my_elements[index]])
```

Comment:
It works, but over-engineering.

It is enough to have 1 loop with enumerate. Please think about logic